sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/csv/CSVSuite.scala

```diff
@@ -345,6 +346,47 @@ class CSVSuite extends QueryTest with SharedSQLContext with SQLTestUtils with Te
     assert(result.schema.fieldNames.size === 1)
   }
 
+  test("SPARK-26339 Debug statement if some of specified paths are filtered out") {
+    class TestAppender extends AppenderSkeleton {
```

Comment:
I wouldn't bother with this complexity to test if the debug log was printed; it's not important compared to the additional binding to log4j.

Reply:
I fixed it.